### PR TITLE
0.9.1, 0.9.2, 0.9.3 Aqua changelog

### PR DESCRIPTION
**Pull request by DieMyst**

### CODE REVIEW

**Comment by DieMyst** [comment]
fixes LNG-110

File: docs/aqua-book/language/types.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ func createStruct(i: []u32) -> SomeStruct:
         num = 4,
         inner = InnerStruct(arr = ["a", "b", "c"], num = i[2])
     )
+    
+-- Also, structures can be copied with arguments replacing
```
Comment (folex) [inline docs/aqua-book/language/types.md]:
```suggestion
-- To modify a structure value, there's a method called 'copy'. It creates a copy with values of specified fields changed.
-- Note that copy operates in an immutable way: it does not modify original structure value.
```

File: docs/aqua-book/language/types.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ func createStruct(i: []u32) -> SomeStruct:
...
+func copyStruct(someStruct: SomeStruct) -> SomeStruct:
```
Comment (folex) [inline docs/aqua-book/language/types.md]:
```suggestion
func modifyStruct(someStruct: SomeStruct) -> SomeStruct:
```

File: docs/aqua-book/changelog.md [hunk 3]
```diff
@@ -7,6 +7,13 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
 * `ENHANCING` part is incremented for every syntax addition
 * `RELEASE` is the release number, shows internal compiler changes, bugfixes that keep the language untouched
 
+### [0.9.2](https://github.com/fluencelabs/aqua/releases/tag/0.9.2) – January 20, 2023
```
Comment (folex) [inline docs/aqua-book/changelog.md]: Maybe add an entry about `copy` to the changelog?

File: docs/aqua-book/changelog.md [hunk 4]
```diff
@@ -7,6 +7,18 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
 * `ENHANCING` part is incremented for every syntax addition
 * `RELEASE` is the release number, shows internal compiler changes, bugfixes that keep the language untouched
 
+### [0.9.3](https://github.com/fluencelabs/aqua/releases/tag/aqua-v0.9.3) – January 20, 2023
+* `copy` method was implemented to modify a structure value ([example](https://fluence.dev/docs/aqua-book/language/types.md#structures), PR: [#646](https://github.com/fluencelabs/aqua/pull/646))
```
Comment (folex) [inline docs/aqua-book/changelog.md]:
```suggestion
* Add `copy` method to modify a structure field while preserving immutability ([example](https://fluence.dev/docs/aqua-book/language/types.md#structures), PR: [#646](https://github.com/fluencelabs/aqua/pull/646))
```

File: docs/aqua-book/changelog.md [hunk 5]
```diff
@@ -7,6 +7,18 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
...
+* bug fix where `nil` cannot be returned from a function properly ([#629](https://github.com/fluencelabs/aqua/pull/629))
```
Comment (folex) [inline docs/aqua-book/changelog.md]:
```suggestion
* bug fix: sometimes `nil` cannot be returned from a function properly ([#629](https://github.com/fluencelabs/aqua/pull/629))
```

File: docs/aqua-book/changelog.md [hunk 6]
```diff
@@ -7,6 +7,18 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
...
+* incorrect number of arguments error message fix ([#645](https://github.com/fluencelabs/aqua/pull/645))
```
Comment (folex) [inline docs/aqua-book/changelog.md]:
```suggestion
* bug fix: incorrect number of arguments error message ([#645](https://github.com/fluencelabs/aqua/pull/645))
```

**Review by folex — APPROVED** [review]

**Pull request merged**